spec/meilisearch/client/indexes_spec.rb

```diff
@@ -4,19 +4,47 @@
   describe '#create_index' do
     context 'without a primary key' do
       it 'creates an index' do
-        index = client.create_index('new_index')
+        task = client.create_index('new_index')
```

Comment:
So cool! did not know about this! since it concerns the whole spec, I've opened an issue: https://github.com/meilisearch/meilisearch-ruby/issues/284
This is too much unrelated to this PR, I would rather a separated PR.

Reply:
I'll apply change regarding the tests I created of course :)